### DISCUSSION TITLE
Drag velocity

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [pmndrs/use-gesture#489] [Q&A]
Hi! I was looking into the instagram carousel and i've noticed that depending on the speed in which the user swipe (fast or slow), the carousel slide accordingly. Using this library with `react-spring`, a similar result can be achieved by tweaking the config object, but this don't reflect the user dragging velocity. Is there a way to get this values to pass them accordingly to react spring? 
 I know that we can get the `velocity` array from the state, but can these be, somehow, translated into spring config?

Thanks a lot! :)

**Comment by dbismut** [comment]
Hey @Emiliano-Bucci not sure how I can help you with this. In theory, you should be able tot pass the velocity when the drag ends to react-spring. Does that not work as intended?
